apps/dashboard/test/integration/pinned_apps_test.rb

```diff
@@ -66,6 +66,14 @@ def teardown
     ], dditems
   end
 
+  test "Files app displays terminal button when configuration is set to true" do
+    Configuration.stubs(:files_app_can_launch_terminal?).returns(true)
+
+    get '/files/fs/tmp'
+
+    assert_select "div[id='shell-wrapper']", 1
+  end
```

Comment:
This should likely go into some other integration test file (one for the files app if we have it?).

Also should test for the inverse, when it's disabled make sure it _doesn't show_.

Reply:
Good call. I'll test both and create a files_app_test.rb.